typeDefs.js

```diff
@@ -190,6 +190,7 @@ const typeDefs = gql`
 
     getAllCategories: [Category]
     getCategoryById(id: ID): CategoryResult
+    getSubcategories(id: ID!): [Category]
```

Comment:
Please, rename param `id` to the `parentCategoryId` to improve readability and understandability of code

Reply:
renamed